### DISCUSSION TITLE
Are startup error messages logged to any files?

### DISCUSSION THREAD

**Discussion by mkastner** [LazyVim/LazyVim#1963] [General]
I have a fresh install of lazy vim and I keep getting error messages on startup (something about macos and also something about treesitter not compiling) but the vanish after a second or two.

Does anybody know, whether these errors are being logged somewhere?

I've tried to find information about this, but either I am looking at the wrong places or my terminology is misleading.

**Comment by amaanq** [comment]
`:mess` or `:Noice log`

**Reply by mkastner** [reply]
Finally, thanks so much!

**Comment by oracleyue** [comment]
I am also fresh and use the current lastest LazyVim starter. I noticed the same issue but found no way to see the full message or log file. Maybe I just do not know something and ask a stupid question. Sorry if so, and thanks a lot for your kindly help!

*To reproduce:*

Add any lua config file in plugins with any mistake in it (like syntax error in lua). Then start nvim, we see the popup window to show the error and stack tracebacks, but `:mess` or `Noice all` show nothing, the default log file of neovim in `~/.local/state/nvim/log` is empty (actually it seems empty forever?). 

- May I know how to get the complete message, which is cut in the popup?
- Has LazyVim disabled the logging by default? How can I turn logging on?

Many thanks!!

e.g.,
<img width="1340" alt="Snipaste_2024-11-16_04-31-19" src="https://github.com/user-attachments/assets/e567acfc-15f6-4063-a8d9-e0558f8227b1">

**Reply by dpetka2001** [reply]
`:lua Snacks.notifier.show_history()`

**Reply by oracleyue** [reply]
> `:lua Snacks.notifier.show_history()`

Wow, thanks soooo much! (I should have asked instead of searching and debugging for 4+ hours. Many many thanks for such a prompt response!)

**Reply by VesperQuartz** [reply]
> > `:lua Snacks.notifier.show_history()`
> 
> Wow, thanks soooo much! (I should have asked instead of searching and debugging for 4+ hours. Many many thanks for such a prompt response!)

It is truncated
